app/screens/channel/channel_post_list/intro/illustration/private.tsx

```diff
@@ -1,280 +1,219 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import * as React from 'react';
-import Svg, {
-    G,
-    Path,
-    Ellipse,
-    Mask,
-    Defs,
-    Pattern,
-    Use,
-    Image,
-    LinearGradient,
-    Stop,
-    ClipPath,
-} from 'react-native-svg';
+import React from 'react';
+import {Svg, G, Path, Circle, Rect, Defs, ClipPath} from 'react-native-svg';
```

Comment:
Nitpick: Any reason to change this import from multiline to single line?

Reply:
My comment was more about avoiding unneeded diffs.

In general I like multiline because it is more clear what is new and what is removed, but I don't have a strong opinion. And as you said, there is no consistency right now, and I don't think there is any need to look for consistency here.

In this particular case, probably was used a multiline approach because we were importing a lot of things and the line got too long. Now that we are importing a smaller set of things from react native svg, it should be fine.